staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -1168,10 +1142,10 @@ func getTenantForKey(segs []string, posToGet int) string {
 
 // getClientsFromKey is used by list to get all related storage clusters
 // For example: get pods from all tenants
-func (s *store) getClientsFromKey(key string) map[uint8]*clientv3.Client {
+func (s *store) getClientsFromKey(key, tenant string) map[uint8]*clientv3.Client {
```

Comment:
Sometimes it is a list that need to go to multiple client. Such as list pods across all tenants.